pwiz_tools/Skyline/TestData/CommandLineRefineTest.cs

```diff
@@ -437,7 +437,8 @@ public void ConsoleArgumentInvalidValuesTest()
             {
                 CommandArgs.ARG_PANORAMA_FOLDER,
                 CommandArgs.ARG_TOOL_INITIAL_DIR,
-                CommandArgs.ARG_TOOL_PROGRAM_PATH
+                CommandArgs.ARG_TOOL_PROGRAM_PATH,
+                CommandArgs.ARG_BGPROTEOME_NAME
```

Comment:
Maybe these should get comments as to why they are special? It is not immediately obvious to me why you need to add yours here.